lib/node_modules/@stdlib/math/base/special/asecf/test/fixtures/julia/runner.jl

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env julia
+#
+# @license Apache-2.0
+#
+# Copyright (c) 2024 The Stdlib Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import JSON
+
+"""
+    gen( domain, name )
+
+Generate fixture data and write to file.
+
+# Arguments
+
+* `domain`: domain
+* `name::AbstractString`: output filename
+
+# Examples
+
+``` julia
+julia> x = range( -1000, stop = 1000, length = 2022 );
+julia> gen( x, \"data.json\" );
+```
+"""
+function gen( domain, name )
+	x = collect( domain );
+	y = acos.( 1.0 ./ x );
```

Comment:
Julia has a dedicated `asec` function. In which case, you should generate test fixtures using that API. While the Julia implementation may, in fact, be `acos(1/x)`, we cannot assume that and we should test against the interface and not the implementation.

Once changed, the text fixtures should be regenerated accordingly.

Reply:
I regenerated the fixtures using `asec`. Tried decreasing the tolerance after the change, but it can't be decreased further, which does makes sense.

The main reason why I had used `acos( 1 / x )` instead of `asec( x )`, was that in our [`asec` implementation](https://github.com/stdlib-js/stdlib/blob/01cbab9ea365ede593f291f90959bbf9f7a8e967/lib/node_modules/%40stdlib/math/base/special/asec/test/fixtures/julia/runner.jl#L40) too, we have done the same. Shall I change that too ?

Reply:
Sigh. Yes, please update the `asec` fixture. That shouldn't have gotten through.

Reply:
Thanks for catching and volunteering to fix!

Reply:
Sure !